c/include/ml-api-common.h

```diff
@@ -69,6 +69,7 @@ typedef enum {
   ML_NNFW_TYPE_NNTR_INF = 12,         /**< Inference supported from NNTrainer, SR On-device Training Framework (Since 6.5) */
   ML_NNFW_TYPE_VD_AIFW = 13,          /**< Inference framework for Samsung Tizen TV (Since 6.5) */
   ML_NNFW_TYPE_TRIX_ENGINE = 14,      /**< TRIxENGINE accesses TRIV/TRIA NPU low-level drivers directly (.tvn). (Since 6.5) You may need to use high-level drivers wrapping this low-level driver in some devices: e.g., AIFW */
+  ML_NNFW_TYPE_MXNET = 15,            /**< Apache MXNet (.params, .json). */
```

Comment:
Please set to `0x2002` and move to under the `ML_NNFW_TYPE_SNAP`.  And add comment sth like "only for Android"

Reply:
To reviewers
This change requires new ACR for Tizen release. There are two options to fix it, 
1. just do ACR process (mxnet only for android)
2. internal nnfw-type control with flag (0x2000 is nnfw for android)

Reply:
Option 1.